File: src/sdk-reference/php/3/collection/set-headers/snippets/set-headers-1.php

<?php

use \Kuzzle\Kuzzle;

$headers = [
  'someContent' => 'someValue'
];

$kuzzle = new Kuzzle('localhost');
$dataCollection = $kuzzle->collection('collection', 'index');
$dataCollection->setHeaders($headers, true);

File: src/sdk/android/3/core-classes/collection/set-headers/snippets/set-headers-1.java

JSONObject headers = new JSONObject()
  .put("someContent", "someValue")
  .put("volatile", new JSONObject()
    .put("someVolatileData", new JSONArray()
      .put("with")
      .put("some")
      .put("values")
    )
  );

kuzzle
  .collection("collection", "index")
  .setHeaders(content, true);
